cmd/nodecmd/ssh.go

```diff
@@ -157,7 +185,13 @@ func sshHosts(hosts []*models.Host, cmd string) error {
 		}
 		if isParallel {
 			for hostID, result := range wgResults.GetResultMap() {
-				ux.Logger.PrintToUser("[%s] %s", hostID, fmt.Sprintf("%v", result))
+				for _, host := range hosts {
```

Comment:
we should do something better to avoid double loop. looks like `func GetHostByNodeID(nodeID string, inventoryDirPath string) (*models.Host, error) ` is not used at all, let's rewrite it to be used for this. Also change name GetHostByNodeID -> GetHostByCloudID